frontend/model/contracts/group.js

```diff
@@ -210,7 +209,7 @@ function isActionNewerThanUserJoinedDate (height: number, userProfile: ?Object):
   if (!userProfile) {
     return false
   }
-  return userProfile.joinedHeight < height
+  return userProfile.status === PROFILE_STATUS.ACTIVE && userProfile.joinedHeight < height
```

Comment:
Since we are not remove profile in the state.profiles when the group member leaves group, we should check the profile status in order to some errors not to be happening unexpectedly. Actually, it was happening.